tests/framework/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
nit: I think this file would be better suited in `resources/tests`

Reply:
I have moved this to the suggested location.